dashboard/src/api/hooks/useGetVersions.ts

```diff
@@ -0,0 +1,19 @@
+import { useQuery } from '@tanstack/react-query';
+import { GetVersionsResponse } from '../../interfaces/versions';
+import client from '../client';
+import '../mocks/useGetVersions.mock';
+
+const getVersions = async (projectId?: number) => {
+  const response = await client.get<GetVersionsResponse>(
+    `/api/v1/projects/${projectId}/versions`,
```

Comment:
make "/api/v1/projects" as an exportable constants, because it is used different places.